webapp/src/pages/Settings.tsx

```diff
@@ -369,6 +390,11 @@ const Settings: React.FC = () => {
                         value={value}
                         disabled={!resultingConfig.uncertainty}
                         inputProps={STEPPER[field]}
+                        InputProps={{
+                          endAdornment: (
+                            <InputAdornment position="end">ms</InputAdornment>
```

Comment:
That should be `{UNITS[field]}`, right?
```suggestion
        endAdornment: <InputAdornment position="end">{UNITS[field]}</InputAdornment>,
```